### ISSUE TITLE
Add munki version requirement to pkgsinfo

### ISSUE THREAD

**Issue by GoogleCodeExporter** [gonchik/munki#71]
```
I'd like to propose the idea of adding a "requires_munki_version" to the 
pkgsinfo's available keys.  The idea being a package in the repo won't be 
installed unless a certain version of munki is already installed on the client. 
 A possible second key to add with this one would be what action to take if the 
munki version isn't sufficient.

The scenario where this would be useful is when a specific package or package 
install feature is only available in a development release of Munki.  An 
example I outline on munki-dev here, 
http://groups.google.com/group/munki-dev/browse_thread/thread/c9e04eb9119a97f7. 
 Basically the need was to keep Adobe Reader 10.0.1 from being installed on 
computers less than munki r1061 since I am using the new preflight pkgsinfo key 
to correctly uninstall version 9 and 10.0.0 of Adobe Reader.

Thanks
- Trey
```

Original issue reported on code.google.com by `treyd...@gmail.com` on 25 Mar 2011 at 5:16

**Comment by GoogleCodeExporter** [comment]
```
I've started to work on a fix for this.  The attached patch splits out the 
get_version function into two functions.  In the future, we'll be able to 
*just* get the build number of the munki-core by calling get_build().  

Not being familiar with exactly how munki is working, I'm wondering where best 
within the source to add the checks to see if the client has the correct build 
(or higher) to install the package.  Can someone point me where to start 
looking?
```

Original comment by `ricky.ch...@gmail.com` on 2 Apr 2011 at 7:31

Attachments:
- [munkicommon_splitting_out_get_build.patch](https://storage.googleapis.com/google-code-attachments/munki/issue-71/comment-1/munkicommon_splitting_out_get_build.patch)

**Comment by GoogleCodeExporter** [comment]
```
Addressed in 0.8.3 Build 1564
```

Original comment by `gregnea...@mac.com` on 7 Jul 2012 at 7:07
- Changed state: **Fixed**
